x-pack/plugins/actions/server/config.ts

```diff
@@ -127,6 +127,15 @@ export type ActionsConfig = TypeOf<typeof configSchema>;
 export function getValidatedConfig(logger: Logger, originalConfig: ActionsConfig): ActionsConfig {
   const proxyBypassHosts = originalConfig.proxyBypassHosts;
   const proxyOnlyHosts = originalConfig.proxyOnlyHosts;
+  const proxyUrl = originalConfig.proxyUrl;
+
+  if (proxyUrl) {
+    try {
+      new URL(proxyUrl);
+    } catch (err) {
+      logger.warn(`The confguration xpack.actions.proxyUrl: ${proxyUrl} is invalid.`);
```

Comment:
I was thinking it might be nice to print the message from the error, but testing from the CLI all it ever seems to return is "Invalid URL", along with `code: "ERR_INVALID_URL"`, so I guess we can't really improve this :-)